crates/core/args.rs

```diff
@@ -346,6 +346,50 @@ impl Args {
         Ok(self.matches().walker_builder(self.paths())?.build())
     }
 
+    /// Sort subjects if a sorter is specified
+    pub fn sort(&self, subjects: Vec<Subject>) -> Vec<Subject> {
+        let sorter = match self.matches().sort_by() {
+            Ok(v) => v,
+            Err(_) => return subjects,
+        };
+        if sorter.check().is_err() {
+            return subjects;
+        }
+        #[inline]
+        fn unzip<K, V>(v: Vec<(K, V)>) -> Vec<V> {
+            v.into_iter().map(|v| v.1).collect()
+        }
```

Comment:
So this isn't really `unzip`. `unzip` is `Vec<(K, V)> -> (Vec<K>, Vec<V>)`. And functions like this shouldn't be marked as `#[inline]`: they are polymorphic so it will get inlined anyway. And even if it weren't, the perf overhead of a function call in this context is negligible.

In any case, I would get rid of this function entirely. Instead, do:

```rust
let items = match sorted.kind { ... };
items.into_iter().map(|v| v.1).collect()
```

Reply:
Yep, changed it.